src/view/components/clue/Clue.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import { MICROBIT_TOOLBAR_ID } from "../toolbar/SensorModalUtils";

```

Comment:
Should we remove the Toolbar from this PR for now? I know we'll fix it soon but having it here makes it kind of messy and we could forget some things.  We can just have a comment `// Todo: toolbar` (pt 1)